CMakeLists.txt

```diff
@@ -9,34 +9,38 @@ file(GLOB PACKET_SOURCES packets/*)
 
 add_library(client OBJECT networking/client.cpp)
 add_library(server OBJECT networking/server.cpp)
+# add_library(physics OBJECT "${PROJECT_SOURCE_DIR}/physics_engine/physics_world.cpp" "${PROJECT_SOURCE_DIR}/physics_engine/player_object.cpp" "${PROJECT_SOURCE_DIR}/physics_engine/collider.cpp")
+add_library(physics OBJECT "${PROJECT_SOURCE_DIR}/game/physics_world.cpp" "${PROJECT_SOURCE_DIR}/game/player_object.cpp" "${PROJECT_SOURCE_DIR}/game/collider.cpp")
 
 link_directories(lib)
 
 file(COPY include/assimp/bin/libassimp-5.dll DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 file(COPY shaders DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 file(COPY art DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 
+file(COPY ${PROJECT_SOURCE_DIR}/shaders DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
 add_executable(server_core game/game_logic_pg.cpp game/server_core.cpp ${PACKET_SOURCES})
 target_include_directories(server_core PRIVATE include)
-target_link_libraries(server_core PRIVATE server client ws2_32)
+target_link_libraries(server_core PRIVATE server client physics ws2_32)
 
 add_executable(client_core game/game_logic_client.cpp game/client_core.cpp ${PACKET_SOURCES} ${GRAPHICS_SOURCES})
 target_include_directories(client_core PRIVATE include include/assimp/include)
-target_link_libraries(client_core PRIVATE server client ws2_32 glfw3 assimp zlibstatic)
+target_link_libraries(client_core PRIVATE server client physics ws2_32 glfw3 assimp zlibstatic)
 
 # ----- Testing executables and tests below -----
 enable_testing()
 
 # ---- Network tests ----
 add_executable(NetworkUnitTests tests/network_test.cpp)
-target_link_libraries(NetworkUnitTests PRIVATE server client ws2_32 mswsock advapi32)
+target_link_libraries(NetworkUnitTests PRIVATE server client physics ws2_32 mswsock advapi32)
 
 add_test(NAME NetworkConnectTest COMMAND NetworkUnitTests "connect")
 add_test(NAME NetworkDataTransportTest COMMAND NetworkUnitTests "data_transport")
 
 # ---- Core tests ----
 add_executable(ServerCoreTests tests/servercore_test.cpp game/server_core.cpp ${PACKET_SOURCES})
-target_link_libraries(ServerCoreTests PRIVATE server client ws2_32)
+target_link_libraries(ServerCoreTests PRIVATE server physics client ws2_32)
```

Comment:
physics isn't necessary unless unit tests were updated to utilize physics. but that should really be a physics unit test at that point (and honestly I don't think it's worth our time to write more tests at this point)